### DISCUSSION TITLE
用了半个月后，我的站点被墙了

### DISCUSSION THREAD

**Discussion by rossroma** [ddiu8081/chatgpt-demo#334] [General]
我是使用vercel部署的，绑定了自己的二级域名，域名备案在阿里云。
之前一直可以正常访问，昨天同步了一下最新代码，并添加了需要密码访问以及google analytics，下午晚些时候，网站就无法在大陆正常访问了。【但我无法确定，网站被ban和我当天的操作是否有联系】

我自己怀疑是网页中包含了“chatgpt”等关键词，被GFW检索到，然后被封了。我记得https://chatgpt.ddiu.me/ 这个域名最早也是无需翻墙访问的，现在也不行了，估计是因为同样的原因被封。

大家使用自己域名绑定的，接下来可能要小心了。我目前只是二级域名被封，不知道接下来会不会波及到主域名。其他人是否有类似的经历，可以一起分享交流下。

![image](https://user-images.githubusercontent.com/6183683/229002248-890a5a95-b707-4886-a4f0-9e6001db63f4.png)

![image](https://user-images.githubusercontent.com/6183683/229002345-b39db075-9500-4291-8f45-4d124e37a294.png)

![image](https://user-images.githubusercontent.com/6183683/229002404-3fc7e3ae-c222-4d94-8370-fe502c4b5191.png)

**Comment by chenyhd** [comment]
把域名转到 cloudflare 上应该会好一些，gpt 本来在国内就是被ban的，阿里云只需要停止对你的  dns 解析就行了，另外 vercel 国内目前无法访问，也有可能是阿里云无法到 vercel 导致的

**Comment by ddiu8081** [comment]
应该和同步代码的操作没太大关系，chatgpt.ddiu.me 是三月初就被墙的

**Comment by kaiyujiang** [comment]
可以学下加密码的代码么

**Comment by nsnsqk** [comment]
vercel 在官网，在2023.9月还可以访问的，现在被墙了。哎，还有哪些可部署的平台吗？

**Reply by iRedScarf** [reply]
今年国庆前的GFW相当激进，可以用自定义域名，托管在CloudFlare，就可以继续使用Vercel了
